cloud-service-providers/aws/eks/README.md

```diff
@@ -0,0 +1,179 @@
+# NIM-EKS: NVIDIA Inference Microservices on AWS EKS
```

Comment:
This should just read "NVIDIA NIM on AWS EKS", no need to spell NIM out.

Reply:
Updated.